music.js

```diff
@@ -1,17 +1,29 @@
 // Define the Artist constructor function here, with a 'name' property that can be set at instantiation,
 // and a 'songs' property that is an empty array to start
-function Artist() {
+function Artist(name) {
+  this.name = name;
+  this.songs = [];
 
 }
 
 // Define the Song constructor function here, with 'title' and 'artist' properties that can be set at instantiation,
 // and a 'playCount' property that is an empty array to start
-function Song() {
-
+function Song(title, artist) {
+  this.title = title;
+  this.artist = artist;
+  playCount = [];
```

Comment:
The playCount should also be `this.playCount` so its value can vary for each instance of Song.